OP-388 - improve docs on area I hit friction on because I am not an e… (#100)
circonus/docs: content/circonus/administration/enterprise-brokers.md

     uncomment the following line, setting the value to your deployment's API
     URL:
     * `CIRCONUS_API_URL="https://api.your.site.domain"`

> 1. Configure optional settings in `noit.local.env`.  A listing of optional parameters is in [Environment Variables](//circonus/administration/enterprise-brokers/#environment-variables) below.  Commonly used parameters are:

Reviewer: ```suggestion
1. Configure optional settings in `noit.local.env`.  A list of optional parameters is in [Environment Variables](//circonus/administration/enterprise-brokers/#environment-variables) below.  Commonly used parameters are:
```

Reviewer: I would argue for including BROKER_IP among "commonly used" parameters but maybe I'm in the minority there.

  from Circonus, as detailed in [External
  Connectivity](/circonus/administration/enterprise-brokers/#external-connectivity)
  above.

> * `BROKER_NAME` : An user-friendly alias for this broker that will be displayed in the Circonus UI.  Spaces are allowed but must be quoted.  e.g. `BROKER_NAME="Friendly Neighborhood Broker"`

Reviewer: ```suggestion
* `BROKER_NAME` : A user-friendly alias for this broker that will be displayed in the Circonus UI.  Spaces are allowed but must be quoted.  e.g. `BROKER_NAME="Friendly Neighborhood Broker"`
```

  of the interface over which remote addresses are reachable.
* `CLUSTER_NAME` : The name of a cluster to join or create. If the named
  cluster already exists, this broker will join it. If it does not exist, a new

>   cluster will be created with this broker as a member.  `Note:` Once a broker is part of a cluster of more than 1 node, it may no longer be changed, it can only be decomissioned to remove it from the cluster.  Spaces are allowed but must be quoted.  e.g. `CLUSTER_NAME="My Cluster Name"` 

Reviewer: ```suggestion
  cluster will be created with this broker as a member.  **Note:** Once a broker is part of a cluster of more than 1 node, it may no longer be changed, it can only be decommissioned to remove it from the cluster.  Spaces are allowed but must be quoted.  e.g. `CLUSTER_NAME="My Cluster Name"` 
```

There will also be, briefly, a report file with a .btt extension, which is what coroner will upload to Circonus, and then remove. If there are .btt files lingering in the traces directory, check that the coroner service is running, and that the necessary network connectivity is available.

### Decommissioning

To decommission a broker, open the main menu and navigate to "Integrations ->
Brokers", then click "View" to go to the detail page for the broker in
question. From the Menu at top right, choose "Decommission Broker".

> **Warning:**

Reviewer: I feel like this warning should be moved up to the top of the section. Decomm is a last-resort option and people need to understand the consequences. It should continue to precede the actual instructions.

>[Reinstallation](/circonus/administration/enterprise-brokers/#reinstallation)
>section above.

To reuse the same machine for a new broker install, all Circonus

> packages broker packages (and dependancies) must be removed and the `/opt/napp` 

Reviewer: ```suggestion
packages (and dependencies) must be removed and the `/opt/napp` 
```

and `/opt/noit` directories deleted after decommissioning it. 
```

> yum remove circonus*

Reviewer: ```suggestion
yum remove 'circonus*'
```

Reviewer: The single quotes are important, protecting the glob from shell expansion.
